packages/common/src/types/sqlRunner.ts

```diff
@@ -103,6 +115,16 @@ export const isBarChartSQLConfig = (
 ): value is BarChartSqlConfig =>
     !!value && value.type === ChartKind.VERTICAL_BAR;
 
+export const isLineChartSQLConfig = (
+    value: SqlRunnerChartConfig | undefined,
+): value is LineChartSqlConfig => !!value && value.type === ChartKind.LINE;
+
+export const isCartesianChartSQLConfig = (
+    value: SqlRunnerChartConfig | undefined,
+): value is CartesianChartSqlConfig =>
+    !!value &&
+    (value.type === ChartKind.LINE || value.type === ChartKind.VERTICAL_BAR);
```

Comment:
and LINE here as well, correct?

Reply:
I think it's there. 

(value.type === ChartKind.LINE || value.type === ChartKind.VERTICAL_BAR);

Tell me if I am missing what you are saying 😄

Reply:
ignore 🤦